### DISCUSSION TITLE
[trino-394]cann't not open web monitor[discovery.uri]

### DISCUSSION THREAD

**Discussion by wc99** [trinodb/trino#13955] [Q&A] [answered]
hello !
When I finished installing the trino service[194], I could no longer open the web address[`http://10.199.1.24:10080/ui`] in my browser。Curiously, the server logs do not have any error information
mysql trino service have three server ，one coordinator and two work
The configuration file for the coordinator is as follows
```
#node.properties
node.environment=trino_dev
node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
node.data-dir=/opt/pi/trino/data

#config.properties 
coordinator=true
node-scheduler.include-coordinator=false
http-server.http.port=10080
query.max-memory=6GB
query.max-memory-per-node=2GB
#query.max-total-memory-per-node=4GB
discovery.uri=http://10.199.1.24:10080
```
The configuration file for the work is as follows
```
#node.properties
node.environment=trino_dev
node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
node.data-dir=/opt/pi/trino/data

#config.properties 
coordinator=true
node-scheduler.include-coordinator=false
http-server.http.port=10080
query.max-memory=6GB
query.max-memory-per-node=2GB
#query.max-total-memory-per-node=4GB
discovery.uri=http://10.199.1.24:10080
```
when trino server start, i use `jps` command  check server status
```
[trino_pi@ha-node1 log]$ jps
30224 TrinoServer
32636 Jps
```
 the coordinator server.log :
[server.zip](https://github.com/trinodb/trino/files/9467340/server.zip)

when i use google open the url `http://10.199.1.24:10080/ui` it show that `ERR_UNSAFE_PORT` 
Please Help me！！！

**Comment by hashhar** [comment, marked as answer]
Browsers mark some ports as unsafe and show warnings when using those ports. Try using a different port and it should work.

See https://jazzy.id.au/2012/08/23/why_does_chrome_consider_some_ports_unsafe.html for context and https://chromestatus.com/feature/6510270304223232

**Reply by wc99** [reply]
I modified the port to another one ， it work now
thank u very much
